myst.yml

```diff
@@ -32,4 +32,7 @@ project:
 site:
   template: book-theme
   options:
+    logo: style/icons/jupyternaut-lite.svg
     logo_text: JupyterLite AI
+    favicon: docs/favicon.ico
```

Comment:
Can't we use the same file for both favicon and logo, as you suggest @jtpio ?

It seems to work well locally.

Reply:
Yes let's do that.

Reply:
Hmm it seems the favicon is not appearing anymore on the preview:

<img width="1110" height="306" alt="image" src="https://github.com/user-attachments/assets/dd74c567-68fc-4217-b6dd-eb100426c907" />

Reply:
Looks like it's still `favicon.ico` being requested:

<img width="916" height="240" alt="image" src="https://github.com/user-attachments/assets/f5f750b2-744b-4ecd-a90e-7b33dd250dc0" />

If it's too complicated to configure with myst, let's just revert to what we had before with the extra `.ico` file?